tfx/components/example_validator/executor.py

```diff
@@ -63,30 +66,40 @@ def Do(self, input_dict: Dict[Text, List[types.Artifact]],
       output_dict: Output dict from key to a list of artifacts, including:
         - output: A list of 'ExampleValidationPath' artifact of size one. It
           will include a single pbtxt file which contains all anomalies found.
-      exec_properties: A dict of execution properties. Not used yet.
+      exec_properties: A dict of execution properties.
+        - exclude_splits: Names of splits that the example validator should not
+          validate.
 
     Returns:
       None
     """
     self._log_startup(input_dict, output_dict, exec_properties)
 
-    absl.logging.info('Validating schema against the computed statistics.')
-    label_inputs = {
-        labels.STATS:
-            tfdv.load_statistics(
-                io_utils.get_only_uri_in_dir(
-                    artifact_utils.get_split_uri(input_dict[STATISTICS_KEY],
-                                                 'eval'))),
-        labels.SCHEMA:
-            io_utils.SchemaReader().read(
-                io_utils.get_only_uri_in_dir(
-                    artifact_utils.get_single_uri(input_dict[SCHEMA_KEY])))
-    }
-    output_uri = artifact_utils.get_single_uri(output_dict[ANOMALIES_KEY])
-    label_outputs = {labels.SCHEMA_DIFF_PATH: output_uri}
-    self._Validate(label_inputs, label_outputs)
-    absl.logging.info(
-        'Validation complete. Anomalies written to {}.'.format(output_uri))
+    exclude_splits = exec_properties.get(EXCLUDE_SPLITS_KEY)
```

Comment:
you can just use exec_properties['EXCLUDE_SPLITS_KEY'] as EXCLUDE_SPLITS_KEY will also be in the exec_properties map, just the value might be None (you have not exclude_splits to cover that below)